scripts/quickstart_run.sh

```diff
@@ -109,6 +114,14 @@ set -e
 # don't fail if already exists
 set +e
 export POSTGRES_POD="$(kubectl get pods --no-headers -o custom-columns=":metadata.name" | grep postgres)"
+
+RETRIES=20
```

Comment:
note: a better solution for this would be to add proper readiness/liveness probes to the postgres deployment